src/services/services.ts

```diff
@@ -6286,7 +6286,7 @@ namespace ts {
                         if (type) {
                             const propertySymbol = typeChecker.getPropertyOfType(type, propertyName);
                             if (propertySymbol) {
-                                result.push(propertySymbol);
+                                result.push(...typeChecker.getRootSymbols(propertySymbol));

```

Comment:
I thought we were avoiding this pattern because there can often be many elements in the list - though I guess in practice this probably doesn't come up all that often.